portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ABTestACSide.testcase

```diff
@@ -315,4 +315,56 @@ definition {
         }
     }
 
+    @description = "Story: LRAC-14070 | Automation ID: LRAC-14575 | Test Summary: After terminating a test and 
+    creating a new one assert the publish button is no longer shown and the delete button deletes the test in AC"
+    @priority = 4
+    test AssertPublishButtonNoLongerShownAndDeleteButtonDeletesTestInAC {
```

Comment:
Hey @manuferreira, I think this test case is very similar to:

- CanTerminateABTestAndCreateNewOne

I think you could put the button assert and deletion in this other test so that we don't have two cases doing basically the same thing. What do you think?

Reply:
https://github.com/liferay-ac/liferay-portal/pull/729/commits/95b73ad83479e21fda1019c055301adbe852df28 and https://github.com/liferay-ac/liferay-portal/pull/729/commits/de4494ed36d7b731323b3f5afaff3edaa1bb28a7